core.md

```diff
@@ -0,0 +1,115 @@
+
+The Council also serves double duty as a mediator for disagreements between other teams. These shall be resolved by discussion, rather than by unanimous applications of the rules, so this is not in the purview of this document.
+
+However, the Council does not have all-encompassing powers. They are the avenue of last resort, and the vast majority of decisions should be made by the team in charge, not by the Steering Council.
+
+The Council has these powers to resolve the 1% of exceptional situations, not the 99%.
+
+#### Maintainers
+
+The Maintainers team consists of all project developers who have push access or technical rights to the project's repositories or infrastructure.
+
+They are the ones keeping the Project's repositories up to date, and ensuring the quality and standards of the Project are upheld, as well as adding new features as the community comes to require them.
+
+Though Maintainers may individually or collectively have different access rights across the project's many repositories or infrastructure services, all Maintainers within the team are considered equal to each other with respect to their rights within the team's governance.
+
+Maintainers may be appointed by other Maintainers, or by application from the person who wishes to join the team.
+
+
+#### Moderators
+
+The Moderation team is composed of all moderators of all community spaces (GitHub, Discord) under the purview of the project.
+
+They have the responsibility of enforcing the policies and rules of each space, as well as the Code of Conduct, and applying disciplinary sanctions to people who violate said rules.
+
+#### Subproject Leads
+
+A special group exists for those designated as "subproject leaders".
+
+The members of this group were assigned from the Maintainers team, and is indeed a strict subset of the Maintainers team.
+
+The role of this team is to take lead of a subproject, [as defined above](#subprojects).
+
+The subproject leader has total purview of their given subproject (repository or task) up to and including their primary goal.
+
+They may:
+* Decide the structure of their team, in whatever capacity is neccessary.
+* Decide who may or may not be on their team.
+* Delegate the power of subproject leader to another person.
+* Make final decisions on the implementation and structure of their subproject.
+* Overrule another Maintainer on the strict matter of their subproject, and no other.
+
+They are not:
+* A project dictator.
+* Unaccountable.
+* Free from scrutiny.
+
+They may not:
+* Make decisions not related to their subproject.
+* Make major architectural decisions relating to the entire Project without the assent of the Steering Council as well as the Maintainers.
+
+Finally, the decisions made by the subproject leader and their team are, under all circumstances, subject to regular scrutiny and final review by the Maintainers and the Steering Council, to ensure the subproject still holds to the Project's values and quality standards.
```

Comment:
IMHO: We should add something like: `However, continuously putting them under question is not allowed. They should be able to do their job and should regularly check in with the rest of the maintainers to collect feedback.`